.gitignore

```diff
@@ -111,3 +111,5 @@ install_manifest.txt
 # vim 
 *.swp
 *.swo
+
+docs
```

Comment:
ok - The doxy output could also go to some other directory. I wouldn't mind to keep the config local either. Though it might be handy for others as well to generate include- and call-graphs.